src/Client/Settings.java

```diff
@@ -2365,30 +2288,39 @@ public static void definePresets(Properties props) {
         "custom",
         getPropBoolean(props, "trigger_alerts_replay", TRIGGER_ALERTS_REPLAY.get("default")));
 
-    REPLAY_BASE_PATH.put("vanilla", "");
-    REPLAY_BASE_PATH.put("vanilla_resizable", "");
-    REPLAY_BASE_PATH.put("lite", "");
-    REPLAY_BASE_PATH.put("default", "");
-    REPLAY_BASE_PATH.put("heavy", "");
-    REPLAY_BASE_PATH.put("all", "");
-    REPLAY_BASE_PATH.put(
-        "custom", getPropString(props, "replay_base_path", REPLAY_BASE_PATH.get("default")));
+    defineStaticPreset(REPLAY_BASE_PATH, getPropString(props, "replay_base_path", ""));
 
     PREFERRED_DATE_FORMAT.put(
-        "vanilla", "dd MMMMMMMMM yyyy - HH:mm:ss"); // jagex is british so this is vanilla
-    PREFERRED_DATE_FORMAT.put("vanilla_resizable", "dd MMMMMMMMM yyyy - HH:mm:ss");
-    PREFERRED_DATE_FORMAT.put("lite", "dd MMMMMMMMM yyyy - HH:mm:ss");
+        "vanilla",
+        getPropString(
+            props,
+            "preferred_date_format",
+            "dd MMMMMMMMM yyyy - HH:mm:ss")); // jagex is british so this is vanilla
+    PREFERRED_DATE_FORMAT.put(
+        "vanilla_resizable",
+        getPropString(props, "preferred_date_format", "dd MMMMMMMMM yyyy - HH:mm:ss"));
     PREFERRED_DATE_FORMAT.put(
-        "default", "yyyy-MM-dd HH:mm:ss"); // ISO 8601, same as default folder name format
+        "lite", getPropString(props, "preferred_date_format", "dd MMMMMMMMM yyyy - HH:mm:ss"));
+    PREFERRED_DATE_FORMAT.put(
+        "default",
+        getPropString(
+            props,
+            "preferred_date_format",
+            "yyyy-MM-dd HH:mm:ss")); // ISO 8601, same as default folder name format
     PREFERRED_DATE_FORMAT.put(
-        "heavy", "MMMMMMMMM dd, yyyy, hh:mm:ss aa"); // american date format for some reason
+        "heavy",
+        getPropString(
+            props,
+            "preferred_date_format",
+            "MMMMMMMMM dd, yyyy, hh:mm:ss aa")); // american date format for some reason
     PREFERRED_DATE_FORMAT.put(
         "all",
-        "EEEEEEE, MMMMMMMMM dd, yyyy GG; hh:mm:ss aa"); // american date format with era and day of
-    // week
+        getPropString(
+            props,
+            "preferred_date_format",
+            "EEEEEEE, MMMMMMMMM dd, yyyy GG; hh:mm:ss aa")); // american format + era + day of week
     PREFERRED_DATE_FORMAT.put(
-        "custom",
-        getPropString(props, "preferred_date_format", PREFERRED_DATE_FORMAT.get("default")));
+        "custom", getPropString(props, "preferred_date_format", "MMMMMMMMM dd, yyyy, hh:mm:ss aa"));
```

Comment:
Not a good change that the default date format is the meme American format, please revert to ISO 8601 as default

Reply:
bad copy paste there, my bad

Reply:
prefer `PREFERRED_DATE_FORMAT.get("default")` since if the default changes (for some ungodly reason), it will automatically be reflected as the default for the Custom setting. This is also acceptable though.